### PR TITLE
Add documentation page for usage stats collection

### PR DESCRIPTION
**Pull request by cw75**

### CODE REVIEW

File: usage.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Usage Stats Collection
+
+As of v0.1.9, Aqueduct collects usage statistics, which helps us identify bugs and better understand
+how to improve our user experience. After starting the server, a prompt will show up informing that
```
Comment (vsreekanti) [inline usage.md:4]: Nit: It's not a prompt, right? We're not asking them to interact with it in anyway. 

I would just say the server start logs will notify you that usage information is being collected.

File: usage.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+usage stats collection is enabled by default. You can disable it by adding `--disable-usage-stats` to `aqueduct start`.
```
Comment (vsreekanti) [inline usage.md:5]: To make it easier to read, make it a dropdown block and write the full command: 

```python
aqueduct start --disable-usage-stats
```

File: usage.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+Please refer to our [usage stats class](https://github.com/aqueducthq/aqueduct/blob/main/src/golang/cmd/server/middleware/usage/models.go) to see what data we collect.
+It is important to note that we do not collect any sensitive or personally identifiable data. We securely
```
Comment (vsreekanti) [inline usage.md:8]: I would move this up front.

File: usage.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+hash your machine ID so that no information related to the original ID will be revealed. We also redact
+any UUIDs related to your workflows, operators, artifacts, etc.
```
Comment (vsreekanti) [inline usage.md:10]: Add a sentence here explaining what we do collect — the endpoints accessed, the latency of the request, etc. etc.

**Comment by cw75** [comment]
@vsreekanti addressed comments

File: usage.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Usage Stats Collection
+
+As of v0.1.9, Aqueduct collects usage statistics, which helps us identify bugs and better understand
```
Comment (vsreekanti) [inline usage.md:3]: typo: which *help* us

File: usage.md (hunk 3 above)
Comment (vsreekanti) [reply usage.md:11]: Maybe add a link to the library we use for this, so the curious can investigate further. You can also link to the code maybe?

**Review by vsreekanti — APPROVED** [review]
One typo, one suggestion line, but LGTM!

**Pull request merged**